pkg/http/infrastructure_handler.go

```diff
@@ -0,0 +1,17 @@
+package http
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+func (h *Handler) Version(c *gin.Context) {
+	version, err := h.api.GetVersion()
+
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"error": "Unable to get blackbeard version"})
```

Comment:
Should'nt you return the error? `err.Error()` could be a part of the return "error". Using `fmt.Sprintf()`